### DISCUSSION TITLE
ESM target support on every matter.js package

### DISCUSSION THREAD

**Discussion by adminy** [matter-js/matter.js#253] [Feature requests (vote via thumbs up)]
matter-node.js package splits its target into esm and cjs, however matter.js package only has common js.

Define a standard way across the packages to target esm. You can import an esm module only one way so maybe it makes sense to stick to one target for simplicity sake but preferably only 1 target for all the packages.

**Comment by Apollon77** [comment]
For the sake of compatibility and integratability into other javaScript projects and libraries I would stay with "a combi package" like matter.js by having cjs and esm builds included and also both exported.
